backend/src/domains/base.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { PrimaryGeneratedColumn, Column, UpdateDateColumn, CreateDateColumn } from 'typeorm';
+
+export abstract class BaseEntity {
+    // @PrimaryGeneratedColumn()
+    // @Column({ type: 'numeric' })
+    // public id: number;
+
+    @CreateDateColumn({ name: 'created_at', type: 'timestamptz', default: () => 'CURRENT_TIMESTAMP' })
+    createdAt: Date;
+
+    @Column({ name: 'created_by', type: 'numeric' })
+    createdBy: number;
+
+    @UpdateDateColumn({ name: 'updated_at', type: 'timestamptz', default: () => 'CURRENT_TIMESTAMP' })
+    updatedAt: Date;
+
+    @Column({ name: 'updated_by', type: 'numeric' })
+    updatedBy: number;
+
+    // @Column({ type: 'boolean', default: true })
```

Comment:
I suppose these fields are not a part of a basic entity, that should be inherited by any other entity in the project. I propose to comment these lines for future using and moving into another generic model or just remove them.

Reply:
I assumed that there would be only one base class. Entities for which these fields are not needed should not inherit this base entity class.

Reply:
But what if some entity will need updated_at and created_at but won't need created_by?